Description: Disable fcf protection in i486 rom builds
 Some of the qemu roms build really old coe with -march=i486 -m16, but
 in this more -fcf-protection isn't available, but recently enabled.
 That causes:
  cc1: error: ‘-fcf-protection’ is not compatible with this target
 Avoid that by disabling the feature in these compile calls until the
 problem is resolved on the toolchain level.
Forwarded: no
X-Not-Forwarded-Reason: Only a problem of the Ubuntu compiler defaults
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1940029
Last-Update: 2021-08-16
--- a/pc-bios/optionrom/Makefile
+++ b/pc-bios/optionrom/Makefile
@@ -13,7 +13,7 @@ CFLAGS = -O2 -g
 quiet-command = $(if $(V),$1,$(if $(2),@printf "  %-7s %s\n" $2 $3 && $1, @$1))
 cc-option = $(if $(shell $(CC) $1 -c -o /dev/null -xc /dev/null >/dev/null 2>&1 && echo OK), $1, $2)
 
-override CFLAGS += -march=i486 -Wall
+override CFLAGS += -march=i486 -Wall -fcf-protection=none
 
 # Flags for dependency generation
 override CPPFLAGS += -MMD -MP -MT $@ -MF $(@D)/$(*F).d
